src/etl/sds/tests/test_sds_etl_components.py

```diff
@@ -24,6 +24,21 @@
 from test_helpers.pytest_skips import long_running
 from test_helpers.terraform import read_terraform_output
 
+# Note that unique identifier "000428682512" is the identifier of 'GOOD_SDS_RECORD'
+DELETION_REQUEST_000428682512 = """
+dn: o=nhs,ou=Services,uniqueIdentifier=000428682512
+changetype: delete
+objectclass: delete
```

Comment:
Im just working my way down the PR so i may get my answer further down - are we making the objectclass delete to trigger something further down? If i recall these are normally the top, changelogentry etc thats the same for all the changelog stuff from ldap

Reply:
ah yeah the hook that was already implemented in the `extract` worker is to filter/hook ldif objects by `objectclass` (i.e. `objectClass: nhsMhs` is mapped on to our python model `NhsMhs`)

This way we can create a new object class `delete` and filter/map it on to the new `SdsDeletionRequest` object

Reply:
and using it as the same value as `changetype` is just convenient, as the same principle can apply to `modify`